packages/types/src/interfaces/fungibles/runtime.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017-2023 @polkadot/types authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { DefinitionsCall } from '../../types/index.js';
+
+export const runtime: DefinitionsCall = {
+  FungiblesApi: [
+    {
+      methods: {
+        query_account_balances: {
+          description: 'Returns the list of all `MultiAsset` that an `AccountId` has',
+          params: [
+            {
+              name: 'account',
+              type: 'AccountId'
+            }
+          ],
+          type: 'Result<Vec<MultiAsset>, FungiblesAccessError>'
```

Comment:
@jacogr 
and I am not sure here with `MultiAsset`, should I import it here?

also here is missing `v3`:  https://github.com/polkadot-js/api/tree/master/packages/types/src/interfaces/xcm
https://github.com/polkadot-js/api/issues/5489

Reply:
The v3 "missing" is expected - all the V3 types are retrieved from the runtime.

I do understand where you are coming from since fixing it to "MultiAsset" means it uses the defined type and not the on-chain type.

(Cannot wait until we have runtime calls inside metadata and all these manual definitions are not required anymore)

Reply:
aha, ok, cool, thank you,

and one more dumb question, is there a way, how could I build and run `polkadot-js.org/apps` with this `api` from this PR locally against my local node? Just to verify, that it works as expected, because I am not sure, if `Vec<MultiAsset>` will work with `XcmVersionedMultiAssets`, because I am also thinking about to change runtime api to use `VersionedMultiAssets`

Reply:
Yes, indeed, you can test locally -

1. Assume you have a folder `<ROOT>`
2. Get both repos into that, i.e. `<ROOT>/apps` and `<ROOT>/api`
3. Make all change to the api
4. Inside the api folder run `yarn polkadot-dev-copy-to apps`
5. It should show it copies stuff
6. Go to the apps folder and start it, `yarn start`
7. Access on `localhost:3000`